File: app/src/main/java/com/kuplay/kuplay/app/SplashActivity.java
package com.kuplay.kuplay.app;

import android.app.Activity;
import android.os.Bundle;
import android.support.annotation.Nullable;
import android.view.View;

import com.kuplay.kuplay.R;
import com.kuplay.kuplay.gdtunion.Splash;

public class SplashActivity extends Activity {

    @Override
    protected void onCreate(@Nullable Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
//        View decorView = getWindow().getDecorView();
//        int uiOptions = View.SYSTEM_UI_FLAG_HIDE_NAVIGATION | View.SYSTEM_UI_FLAG_FULLSCREEN;
//        decorView.setSystemUiVisibility(uiOptions);
//        startActivity(new Intent(this, MainActivity.class));
//        finish();
        View view = this.getWindow().getDecorView();
        view.setBackgroundResource(R.drawable.splash);

        Splash ad = new Splash();
        ad.addSplashAD(this, MainActivity.class);
    }
}

File: app/src/main/java/com/kuplay/kuplay/SplashActivity.kt
package com.kuplay.kuplay

import android.app.Activity
import android.content.Intent
import android.os.Bundle
import android.view.View
import com.kuplay.pi_framework.framework.WebViewActivity

class SplashActivity : Activity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        if (!this.isTaskRoot && intent != null) {
            val action = intent.action
            if (intent.hasCategory(Intent.CATEGORY_LAUNCHER) && Intent.ACTION_MAIN == action) {
                finish()
                return
            }
        } else {
            super.onCreate(savedInstanceState)
            val decorView = window.decorView
            val uiOptions = View.SYSTEM_UI_FLAG_HIDE_NAVIGATION or View.SYSTEM_UI_FLAG_FULLSCREEN
            decorView.systemUiVisibility = uiOptions
            startActivity(Intent(this, WebViewActivity::class.java))
            finish()
        }
    }
}